aks-node-controller/proto/aksnodeconfig/v1/kubelet_config.proto

```diff
@@ -5,6 +5,7 @@ option go_package = "github.com/Azure/agentbaker/aks-node-controller/pkg/gen/aks
 
 message KubeletConfig {
   // A list of taints to apply to the node.
+  // These are reserved for now. We are using KubeletFlags with flag "--register-with-taints" to register node with taints.
   repeated Taint taints = 1;
```

Comment:
Is it currently in use?

Reply:
It's not. Setting it won't be processed by AKSNodeConfig